src/Microsoft.Health.Dicom.Functions.Client/DicomAzureFunctionsClient.cs

```diff
@@ -146,17 +147,68 @@ public async Task<OperationReference> StartExportAsync(Guid operationId, ExportS
     }
 
     /// <inheritdoc/>
-    public Task<Guid> StartBlobCopyAsync(CancellationToken cancellationToken = default)
+    public async Task StartBlobCopyAsync(Guid operationId, CancellationToken cancellationToken = default)
     {
-        // TODO: Implementation is in different PR
-        throw new NotImplementedException();
+        var existingInstance = await _durableClient.GetStatusAsync(operationId.ToString(OperationId.FormatSpecifier), showInput: true);
+
+        if (existingInstance == null)
+        {
+            _logger.LogDebug("No existing copy operation.");
+        }
+        else
+        {
+            _logger.LogDebug("Existing copy operation is in status: '{Status}'", existingInstance.RuntimeStatus);
+        }
+
+        if (IsOperationInterruptedOrNull(existingInstance))
```

Comment:
I would move all of this stuff out of the client. Let the client just be responsible for starting, and let the caller sort out whether it needs to re-submit with the last state. None of that is specific to the implementation of the operations client

Reply:
However, perhaps this method should optionally be given a data range

Reply:
Agree, will change